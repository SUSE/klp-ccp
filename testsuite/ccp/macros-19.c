#include "macros-19.h"

void pu_f(void)
{
	G();
}
