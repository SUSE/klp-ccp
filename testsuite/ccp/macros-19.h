
static void G(void)
{}

#define G g

void G(void)
{}
