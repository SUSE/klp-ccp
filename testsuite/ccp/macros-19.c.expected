/* klp-ccp: from testsuite/ccp/macros-19.h */
#define G g

static void G(void);

/* klp-ccp: not from file */
#undef G
/* klp-ccp: from testsuite/ccp/macros-19.c */
#include "macros-19.h"

void klpp_pu_f(void)
{
	G();
}
